src/ui/popup.js

```diff
@@ -542,6 +548,28 @@ export default class Popup extends Evented {
         applyAnchorClass(this._container, anchor, 'popup');
     }
 
+    _focusFirstElement() {
+        if (!this._container) return;
+
+        // This approach isn't covering all the quirks and cases but it should be good enough.
+        // If we would want to be really thorough we would need much more code, see e.g.:
+        // https://github.com/angular/components/blob/master/src/cdk/a11y/interactivity-checker/interactivity-checker.ts
+        const selectors = [
```

Comment:
There's no need to allocate this array every time this function is called. We could move it out of the function. You could also save it as a `querySelector` to avoid `join`ing the array every time.

Reply:
I moved it to the top-level scope - leaving the `[item1, item2].join(', ')` for better readability.
Let me know if that's fine with you - I can change it to whatever you think works better with the codebase.
I did try one long string, but it was around 190 characters which is just too much, and two strings on two lines didn't look right to me.